docs/en/observability/quickstarts/monitor-hosts-with-otel.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[quickstart-monitor-hosts-with-otel]]
+= Quickstart: Monitor hosts with OpenTelemetry
+
+preview::[]
+
+In this quickstart guide, you'll learn how to collect logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.
+You'll also learn how to get value out of your observability data after collecting it.
+
+[discrete]
+== Prerequisites
+
+* An {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data. This quickstart is available for all Elastic deployment models. To get started quickly, try out our hosted {ess} on {ess-trial}[{ecloud}].
+* A Linux or MacOS system.
+* A user with the **Admin** role or higher—required to onboard system logs and metrics. To learn more, refer to {cloud}/ec-user-privileges.html[User roles and privileges].
+* Root privileges on the host—required to run the OpenTelemetry collector.
```

Comment:
Should we specify why root privileges are required? The two components that require privilege access are:
 -  `hostmetrics` receiver: to read all system metrics (all processes, memory, etc.)
 - `filelog`: so the collector can read any users/application log file